Tests/Functional/JobExecutorInlineConfigTest.php

```diff
@@ -478,122 +477,6 @@ public function testRunTag()
         $this->assertTrue($handler->hasWarning('Overriding component tag with: \'1.1.1\''));
     }
 
-    public function testSandbox()
```

Comment:
neměl by tady přibejt `debugTest`?

Reply:
proc? je to vsechno tady https://github.com/keboola/docker-bundle/blob/odin-debug-api/Tests/Functional/DebugModeTest.php

Reply:
aha, promiň, ten mi utekl :-)

![image](https://user-images.githubusercontent.com/497675/38352840-c367e356-38b4-11e8-8b5d-2e4d15b8d8ca.png)